test/verify/check-storage-scaling

```diff
@@ -35,7 +35,7 @@ class TestStorageScaling(storagelib.StorageCase):
         b.wait_visible(self.card_row("Storage", name="/dev/vda"))
         # Wait on btrfs subvolumes on OS'es with the install on btrfs
         if m.image.startswith('fedora'):
-            b.wait_in_text(self.card_row("Storage", name="root/var/lib/machines"), "btrfs subvolume")
+            b.wait_in_text(self.card_row("Storage", name="var/lib/machines"), "btrfs subvolume")
```

Comment:
With the strip prefix change here this should be the same wait condition for Arch and Fedora `var/lib/machines`:

```
[root@archlinux ~]# btrfs subvolume list -ap /
ID 256 gen 50 parent 5 top level 5 path var/lib/portables
ID 257 gen 50 parent 5 top level 5 path var/lib/machines
```

Reply:
Ah, fixed!